lib/containers/artist.js

```diff
@@ -74,13 +81,18 @@ class Artist extends React.Component {
   render() {
     const windowDimensions = Dimensions.get('window');
     const commonPadding = windowDimensions.width > 700 ? 40 : 20;
+    const displayTabView = this.availableTabs().length > 1;
+
     return (
       <ScrollView scrollsToTop={true}>
         <View style={{ paddingLeft: commonPadding, paddingRight: commonPadding }}>
           <Header artist={this.props.artist} />
-          <TabView titles={this.availableTabs()} selectedIndex={this.state.selectedTabIndex} onSelectionChange={this.tabSelectionDidChange}>
-            {this.renderTabViewContent()}
-          </TabView>
+          {
+            displayTabView ?
+            <TabView titles={this.availableTabs()} selectedIndex={this.state.selectedTabIndex} onSelectionChange={this.tabSelectionDidChange}>
+              { this.renderTabViewContent() }
+            </TabView> : null
+          }
```

Comment:
The works themselves should still be shown here: https://github.com/artsy/emission/pull/159/files#diff-f64004b75dbb599d0061fe4024381638R76